dir.props

```diff
@@ -75,16 +75,17 @@
   <!-- Setup the default output and intermediate paths -->
   <PropertyGroup>
     <BaseOutputPath>$(ProjectDir)bin\</BaseOutputPath>
-    <BaseOutputPathWithConfig>$(BaseOutputPath)$(Configuration)\</BaseOutputPathWithConfig>
-    <BaseOutputPathWithConfig Condition="'$(Platform)' != 'AnyCPU'">$(BaseOutputPath)$(Platform)\$(Configuration)\</BaseOutputPathWithConfig>
+    <Error Condition="'$(OS)' == ''">Need OS</Error>
```

Comment:
Could you make this a little more descriptive?  e.g "The 'OS' Property must be defined?